packages/experimental/Checkbox/src/Checkbox.tsx

```diff
@@ -34,15 +34,16 @@ export const Checkbox = compose<CheckboxType>({
     // now return the handler for finishing render
     return (final: CheckboxProps) => {
       const { label, required, ...mergedProps } = mergeProps(Checkbox.props, final);
+      const labelComponent = (
+        <React.Fragment>
+          <Slots.label key="label">{label}</Slots.label>
+          {!!required && <Slots.required>{typeof required === 'string' ? required : '*'}</Slots.required>}
```

Comment:
just checking my understanding; is !! used here because ```required``` may be a string?

Reply:
Yeah. Specifically it should also avoid bad behavior if the string is empty, which can cause crashes.